owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/security/PassCodeViewModel.kt

```diff
@@ -51,11 +56,20 @@ class PassCodeViewModel(
         return isValid
     }
 
-    private fun loadPinFromOldFormatIfPossible(): String {
-        var pinString = ""
-        for (i in 1..4)
-            pinString += preferencesProvider.getString(PassCodeActivity.PREFERENCE_PASSCODE_D + i, null)
+    fun getNumberOfPassCodeDigits(): Int {
+        val numberOfPassCodeDigits = contextProvider.getInt(R.integer.passcode_digits)
+        return if (numberOfPassCodeDigits >= 4) numberOfPassCodeDigits else 4
```

Comment:
Let's create a new constant `PASSCODE_MIN_LENGTH` instead of having a hardcoded 4 👍